web/modules/custom/dpl_event/src/Access/EventSeriesTabAccessCheck.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\dpl_event\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Provides an access checker for the event series tab.
+ *
+ * This access check ensures that the tab for editing event instances
+ * is only available when there are more than one instance in the series.
+ */
+class EventSeriesTabAccessCheck implements AccessInterface {
+
+  /**
+   * The entity type manager service.
+   */
+  public function __construct(
+    private EntityTypeManagerInterface $entityTypeManager
+  ) {}
+
+  /**
+   * Access callback for the event series edit instances tab.
+   *
+   * Checks if there are more than one instance in the event series and grants
+   * or denies access based on this condition.
+   *
+   * @param \Drupal\Core\Routing\RouteMatchInterface $route
+   *   The route match interface to get route parameters.
+   *
+   * @return \Drupal\Core\Access\AccessResult
+   *   The access result, allowing access if there are more than one instance,
+   *   otherwise forbidding it.
```

Comment:
Consider leaving out these parameters comments.

They are not required by our coding style and if they do not add any value they are just unecessary lines of code.

Reply:
> Currently they are required in phpcs tooling. Not sure if i should change that, so i left them in

OK. I am a bit surprised by this. The next time you run into this please let me know.